Add auto completion intruction to docs (#101)
elastic/ecctl: docs/ecctl-getting-started.asciidoc

🍺  /usr/local/Cellar/ecctl/1.0.0: 6 files, 13MB, built in 7 seconds
----

> NOTE: To get autocompletions working make sure to run `source <(ecctl generate completions)` and to your shell interpreter configuration file by running `echo "source <(ecctl generate completions)" >> ~/.bash_profile` or `echo "source <(ecctl generate completions)" >> ~/.zshrc`.

Reviewer: ```suggestion
NOTE: To get autocompletions working make sure to run `source <(ecctl generate completions)`. If you prefer to add to your shell interpreter configuration file run `echo "source <(ecctl generate completions)" >> ~/.bash_profile` or `echo "source <(ecctl generate completions)" >> ~/.zshrc`.
```

Reviewer 2: ++ on the suggested edit from @karencfv. The only other thing to, er, note is that we should `NOTE:` sparingly. We overuse callouts and people start tuning them out. The type of content you have here looks more like something that might into a `Before you begin` section to be consistent with the rest of our ECE and ESS docs.
